packages/components/src/community/sidebar-community/sidebar-community.tsx

```diff
@@ -18,21 +21,24 @@ type Props = {
     membersCount: number
     imageUrl: string
   }
-  channels?: ChannelGroup[]
+  channels?: ChannelGroupType[]
   selectedChannelId?: string
   onChannelPress: (channelId: string) => void
 }
 
-const Sidebar = (props: Props) => {
+const SidebarCommunity = (props: Props) => {
   const {
     community,
     channels = CHANNEL_GROUPS,
     selectedChannelId,
-    onChannelPress,
+    // onChannelPress,
   } = props
 
   const { name, description, membersCount, imageUrl } = community
 
+  const [value, setValue] = useState(['Welcome'])
+  console.log('SidebarCommunity > value:', value)
```

Comment:
Forgotten console.log?

Reply:
done in https://github.com/status-im/status-web/pull/370/commits/13fdfbe3da72bb8c86176eef4568792058c9a675